pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -43,6 +43,7 @@ func TestFeatureToggleFiles(t *testing.T) {
 
 	ownerlessFeatures := map[string]bool{
 		"prometheusAzureOverrideAudience": true,
+		"azLegacyTemplateVariables":       true,
```

Comment:
^^ I added it to the ownerless section since I don't know who/if it is still real

Interesting that both unclaimed flags are for azure